### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by adamzolyak**
Update docs for clarity and correctness. Also, adds a test run on PR.

### CODE REVIEW

File: .github/workflows/validate.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Validate
-on: [push]
+on: [push, pull_request]
```
Comment (anshulsahni) [inline .github/workflows/validate.yaml:2]: push and pull_request will become redundant

File: README.md [hunk 2]
```diff
@@ -98,10 +102,11 @@ jobs:
...
 
 ## Maintainers
-This repo is mainted by:
 
-* Adam Zolyak @adamzolyak
-* Anshul Sahni @anshulsahni
+This repo is maintained by:
+
+- Adam Zolyak @adamzolyak
+- Anshul Sahni @anshulsahni
```
Comment (anshulsahni) [inline README.md:109]: thanks for this 👍🏼

File: README.md [hunk 3]
```diff
@@ -65,7 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Label PRs
-        uses: tinkurlab/monorepo-pr-labeler-action
+        uses: tinkurlab/monorepo-pr-labeler-action@master
```
Comment (anshulsahni) [inline README.md:72]: we should mention latest version here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -83,6 +69,18 @@ Here are a few things you can do that will increase the likelihood of your pull
 
 Work in Progress pull requests are also welcome to get feedback early on, or if there is something blocked you.
 
+## Releasing
+
+In order to release a new version, you need to compile the package using `npm run build`. This will output a `dist/index.js` file.
+Then, perform the following actions:
+
+- `git add dist/index.js`
+- `git commit -m "Release x.x.x"`
```
Comment (anshulsahni) [inline CONTRIBUTING.md:78]: in future, we should consider integrating [committzen](https://www.google.com/search?channel=fs&client=ubuntu&q=committizen)

**Review by anshulsahni — COMMENTED** [review]
added few very minor comments

File: README.md (hunk 3 above)
Comment (adamzolyak) [reply README.md:72]: "we should mention latest version here" as in the specific version ex `3.1.1`?

File: CONTRIBUTING.md (hunk 4 above)
Comment (adamzolyak) [reply CONTRIBUTING.md:78]: Great idea. Want to add an issue?

**Pull request merged**